wds-network-require-login.php

```diff
@@ -229,20 +250,37 @@ public function maybe_auth_redirect() {
 	 * @return null
 	 */
 	public function auth_redirect() {
+		/**
+		 * Filter the whitelist for the requested URL.
+		 *
+		 * @since 0.1.0
+		 *
+		 * @param array $whitelist The array of URLs to whitelist.
+		 */
 		$whitelist   = apply_filters( 'wds_network_require_login_whitelist', array() );
 		$whitelisted = in_array( $this->current_url, $whitelist );
 
+		/**
+		 * Filter the whitelist for the requested path.
+		 *
+		 * @since 0.2.0
+		 *
+		 * @param array $whitelist The array of paths to whitelist.
+		 */
+		$path_whitelist   = apply_filters( 'wds_network_require_login_path_whitelist', array() );
```

Comment:
Let's only do the existing "wds_network_require_login_whitelist" filter if we got past the "wds_network_require_login_path_whitelist" filter check. Minimal perf. gain, but no reason not to do it.